addons/point_of_sale/static/tests/pos/tours/chrome_tour.js

```diff
@@ -164,8 +164,11 @@ registry.category("web_tour.tours").add("test_tracking_number_closing_session",
             Dialog.confirm("Open Register"),
             ProductScreen.clickDisplayedProduct("Desk Pad", true, "1.0"),
             ProductScreen.clickPayButton(),
-            PaymentScreen.clickPaymentMethod("Bank"),
+            PaymentScreen.clickPaymentMethod("Bank", "20"),
```

Comment:
```suggestion
            PaymentScreen.clickPaymentMethod("Bank"),
```

check the signature of the method:

```js
export function enterPaymentLineAmount(lineName, keys, isCheckNeeded = false, options = {})
```